### ISSUE TITLE
TypeError due to str/bytes concat

### ISSUE THREAD

**Issue by mweinelt** [RealDolos/volapi#13]
```py3tb
[...] volapi/volapi.py", line 119, in send_ack
    self.send_message(b'4' + to_json([self.proto.max_id]))                                                                                                                                                                                                                            
TypeError: can't concat str to bytes  
```

This happens in a few places. Solution is to first encode the json string as a utf-8 bytestring.

```
self.send_message(b'4' + to_json([self.proto.max_id]).encode())
```

Could probably also use something fancy like bytearrays.

**Comment by szero** [comment]
Right, this happens because I introduced new json library, orjson. It only returns bytes objects while python's default one only returns strings.

**Comment by mweinelt** [comment]
True, as you might have seen from my previous `pip freeze` I'm not using orjson. The install on NixOS doesn't work yet and I haven't had time to debug it yet.

**Comment by szero** [comment]
fixed in 047609a
